tests/integration/components/o-s-s/currency-input-test.ts

```diff
@@ -16,6 +16,23 @@ module('Integration | Component | o-s-s/currency-input', function (hooks) {
     assert.dom('.currency-input-container').exists();
   });
 
+  test('it renders currency only', async function (assert) {
+    this.onChange = () => {};
+    this.value = 0;
+    this.currency = '';
+    this.onlyCurrency = true;
+    await render(hbs`<OSS::CurrencyInput @onlyCurrency={{this.onlyCurrency}} @onChange={{this.onChange}} />`);
+
+    assert.dom('.currency-input-container').exists();
+    assert.dom('.currency-selector ').exists();
+
+    const currencyLabel = await findAll('.currency-selector span');
+    assert.dom(currencyLabel[1]).exists();
+    assert.dom(currencyLabel[1]).hasText('USD');
+
+    assert.dom('.currency-input input ').doesNotExist();
+  });
+
```

Comment:
You should also write a test to check that onChange is called with the proper values when currencyOnly is true